packages/mail/express/mailchimp.js

```diff
@@ -179,3 +184,13 @@ const getGroups = (name, data) => {
     .map((g) => g.trim())
     .filter(Boolean)
 }
+
+const applyEnforceSubscriptions = async (record, pgdb) => {
+  try {
+    const { email } = record
+    const user = await pgdb.public.users.findOne({ email })
```

Comment:
Is there any specific reason why not using `findOneFieldOnly` to get userId directly

Reply:
No, indeed, this could be `findOneFieldOnly`.